doc/man/man8/daos.8

```diff
@@ -0,0 +1,174 @@
+.SH NAME
+daos \- command to manage DAOS pool/container/object
+.SH SYNOPSIS
+.br
+.B daos RESOURCE COMMAND \fR[\fBOPTION\fRs]
+.SH DESCRIPTION
+.B daos
+can be used to manage/query a pool content, create/query/manage/destroy a
+container inside a pool or query/manage an object inside a container.
+.br
+The \fBRESOURCE\fRs, respective \fBCOMMAND\fRs and \fBOPTION\fRs supported by \fBdaos\fR are listed below.
+.SH RESOURCE\fRs/\fBCOMMAND\fRs/\fBOPTION\fRs
+.br
+.TP
+.B RESOURCE\fRs:
+	  \fBpool\fR             pool
+.br
+	  \fBcontainer \fR(\fBcont\fR) container in a pool
+.br
+	  \fBobject \fR(\fBobj\fR)     object in a container
+.br
+	  \fBhelp\fR             print this message and exit
+.TP
+.I pool \fBCOMMAND\fRs:
+	  \fBlist-containers\fR  list all containers in pool
+.br
+	  \fBlist-cont\fR
+.br
+	  \fBquery\fR            query a pool
+.br
+	  \fBstat\fR             get pool statistics
+.br
+	  \fBlist-attrs\fR       list pool user-defined attributes
+.br
+	  \fBget-attr\fR         get pool user-defined attribute
+.br
+.TP
+.I pool \fBOPTION\fRs:
+	  \fB--pool=\fRUUID        pool UUID
+.br
+	  \fB--sys-name=\fRSTR     DAOS system name context for servers ("daos_server")
+.br
+	  \fB--sys=\fRSTR
+.br
+	  \fB--svc=\fRRANKS        pool service replicas like 1,2,3
+.br
+	  \fB--attr=\fRNAME        pool attribute name to get
+.br
+.TP
+.I container \fR(\fIcont\fR) \fBCOMMAND\fRs:
+	  \fBcreate\fR           create a container
+.br
+	  \fBdestroy\fR          destroy a container
+.br
+	  \fBlist-objects\fR     list all objects in container
+.br
+	  \fBlist-obj\fR
+.br
+	  \fBquery\fR            query a container
+.br
+	  \fBstat\fR             get container statistics
+.br
+	  \fBlist-attrs\fR       list container user-defined attributes
+.br
+	  \fBdel-attr\fR         delete container user-defined attribute
+.br
+	  \fBget-attr\fR         get container user-defined attribute
+.br
+	  \fBset-attr\fR         set container user-defined attribute
+.br
+	  \fBcreate-snap\fR      create container snapshot (optional name)
+.br
+				    at most recent committed epoch
+.br
+	  \fBlist-snaps\fR       list container snapshots taken
+.br
+	  \fBdestroy-snap\fR     destroy container snapshots
+.br
+				    by name, epoch or range
+.br
+	  \fBrollback\fR         roll back container to specified snapshot
+.TP
+.I container \fBOPTION\fRs (create by UUID):
+	  <\fIpool\fR options>   (\fB--pool\fR, \fB--sys-name\fR, \fB--svc\fR)
+.br
+	  \fB--cont=\fRUUID      (optional) container UUID (or generated)
+.TP
+.I container \fBOPTION\fRs (create and link to namespace path):
+	  <\fIpool\fR/\fIcont\fR opts>   (\fB--pool\fR, \fB--sys-name\fR, \fB--svc\fR, \fB--cont\fR [optional])
+.br
+	  \fB--path=\fRPATHSTR     container namespace path to be created and provide a direct link to new DAOS container
+.br
+	  \fB--type=\fRCTYPESTR    container type (HDF5, POSIX)
+.br
+	  \fB--oclass=\fROCLSSTR   container object class
+.br
+				      (tiny, small, large, R2, R2S, repl_max)
```

Comment:
we should update those to what is supported today:
(ex: S{1,2,4,X}, RP_2G{1,2,4,X}, RP_3G{1,2,4,X}, RP_4G{1,2,4,X}, RP_XSF)

Reply:
Done, but what do all these acronyms mean ??